apps/docs/contentlayer.config.js

```diff
@@ -68,8 +69,45 @@ export default makeSource({
     mdx: {
         remarkPlugins: [],
         rehypePlugins: [
-            [rehypePrettyCode, rehypeOptions]
+            () => tree => {
```

Comment:
What is the purpose of this configuration? Can you explain it to me?

Reply:
Basically we need to catch the "raw" code sample before Rehype transforms it, otherwise we would be copying span's and other transformations that Rehype makes. I can explain in more detail if needed.

Reply:
It's not necessary, I understand the principle.